app/src/test/java/org/mozilla/fenix/components/toolbar/DefaultBrowserToolbarMenuControllerTest.kt

```diff
@@ -64,6 +64,7 @@ import org.mozilla.fenix.utils.Settings
 
 @OptIn(ExperimentalCoroutinesApi::class)
 @RunWith(FenixRobolectricTestRunner::class)
+@Suppress("ForbiddenComment")
```

Comment:
What is ForbiddenComment for?

Reply:
These are for the TODOs that I added :) detekt doesn't like comments